src/Node/Expression/TempNameExpression.php

```diff
@@ -41,7 +41,7 @@ public function __construct(string|int|null $name, int $lineno)
     public function compile(Compiler $compiler): void
     {
         if (null === $this->getAttribute('name')) {
-            $this->setAttribute('name', \sprintf('_l%d', $compiler->getVarName()));
+            $this->setAttribute('name', \sprintf('_l%s', $compiler->getVarName()));
```

Comment:
```suggestion
            $this->setAttribute('name', $compiler->getVarName());
```

Should be enough, right?

Reply:
Probably, I wasn't sure if it was intended to introduce some kind of "namespace" here.

Reply:
code updated as suggested, let's see if the tests agree